Tests/ObservableConverterTests/ObservableConverterTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+@testable import ObservableConverter
+
+final class ObservableConverterTests: XCTestCase {
+    func testExampleConversion() throws {
+        let before = Bundle.module.path(forResource: "ContentView-before", ofType: "swift", inDirectory: "Resources")
+        let after = Bundle.module.url(forResource: "ContentView-after", withExtension: "swift", subdirectory: "Resources")
+        
+        let fileToConvertPath = try XCTUnwrap(before)
+        let afterURL = try XCTUnwrap(after)
+        
+        let testingFileToConvertPath = fileToConvertPath.appending(".testing")
+        try FileManager.default.copyItem(atPath: fileToConvertPath, toPath: testingFileToConvertPath)
+        
+        let converterURL = URL(fileURLWithPath: FileManager.default.currentDirectoryPath.appending("/ObservableConverter"))
```

Comment:
I wish `NSString`’s `appendingPathExtension` and `appendingPathComponent` worked without casting. Anyway, no suggestions here 😄